test/test_jobs/test_delete_tasks.py

```diff
@@ -78,8 +78,8 @@ def test_delete_bulk_tasks_in_batches(self, mock_send_mail, mock_db, mock_cleanu
             'force_reset': True, 'coowners': [], 'current_user_fullname': "usera", 'url': "https://a.com"
         }
 
-        with patch.dict(self.flask_app.config, {"DELETE_BULK_TASKS_IN_BATCHES": True}):
-            delete_bulk_tasks(data) #delete_bulk_tasks_in_batches(data)
+        with patch.dict(self.flask_app.config, {"SESSION_REPLICATION_ROLE_DISABLED": True}):
+            delete_bulk_tasks(data)
```

Comment:
maybe ensure sql contains `SET session_replication_role TO replica;` or `delete_bulk_tasks_with_session_repl` is called in test case.

Reply:
This is covered in another test case [here](https://github.com/bloomberg/pybossa/blob/main/test/test_jobs/test_delete_tasks.py#L44)